src/test/functional/features/final-order.feature

```diff
@@ -25,6 +25,23 @@ Feature: Final order
     When I click "Continue"
     Then the page URL should be "/hub-page"
 
+    Given I set the case state to "FinalOrderRequested"
+    And I click "Sign out"
+    When I login with applicant "1"
+    Then the page should include "You have applied for a ‘final order’. Your application will be checked by court staff."
+    And the page should include "You should receive an email within 2 working days,"
+    Given I click "Sign out"
+    When I login with applicant "2"
+    Then the page should include "Your wife has applied for a ‘final order’. The application will be checked by court staff."
```

Comment:
You could just do the check for app2 first and then sign out and check with app1, saves signing in and out twice

Reply:
Fixed the issue as suggested by Lara